pages/services.js

```diff
@@ -94,6 +95,28 @@ const Services = () => (
         ]}
       />
     </TableOfContents>
+
+    <CTASection
+      className="bx--no-gutter"
```

Comment:
@diegomarcuz Could you please remove this code? The `bx--no-gutter` class only works under divs with `bx--row` class

Reply:
thanks so much @caiobozato. I have just removed.